packages/app/src/debug/utils/DebugMapping.ts

```diff
@@ -0,0 +1,27 @@
+export const specsList = <S extends {id: string}, T extends {specId: string}>(specs: readonly S[], tests: readonly T[]): {spec: S, tests: T[]}[] => {
```

Comment:
See other comment - might not be practical, if so ignore, but it sure would be nice to move this to the backend and have GraphQL serve it up in the format we want.

Reply:
Yes, I think this makes sense as well.  Looking at doing that in the final IATR M0 issue.